pkg/debug/scripts/collect-info.sh

```diff
@@ -44,18 +47,17 @@ DIR="$TMP_DIR/$INFO_DIR"
 mkdir -p "$DIR"
 mkdir -p "$DIR/network"
 
-# Install GNU version of the 'ps' and the dmidecode tool
-echo "- install GNU tools"
-apk add procps dmidecode >/dev/null 2>&1
+# Check for internet access, timeouts at 30 seconds,
+# If there is internet access, install required GNU utilities and networking tools.
+if nc -z -w 30 dl-cdn.alpinelinux.org 443 2>/dev/null; then
+  echo "- install GNU utilities and networking tools"
+  # shellcheck disable=SC2086
```

Comment:
Have to disable SC2086, otherwise it complains about quoting the variable and if we quoting the `$PKG_DEPS` , it gets passed to `apk` like this : 
```
apk add 'procps dmidecode iptables dhcpcd'
ERROR: 'procps dmidecode iptables dhcpcd' is not a valid world dependency, format is name(@tag)([<>~=]version)
```